### ISSUE TITLE
Invalid configuration for 'custom_rules'

### ISSUE THREAD

**Issue by LukasKorba** [realm/SwiftLint#545]
Hi,
seems like there is a misunderstanding of '\' yml versus swift syntax. When I provide a correct rule from the swift perspective, then the yml returns invalid config.. when I use the syntax to satisfy yml, then the swiftlint can't catch my regex.. 

example: 
swift NSRegularExpresion (?:\\S([ \t]|)+\\{) works fine, but yml returns invalid rule
yml version (?:\S([ \t]|)+\{) compile ok, but the xcode doesn't catch the issue.. 

what should I do now? is there some other way how to provide custom rule?

**Comment by norio-nomura** [comment]
As I tested, it works:
```sh
% cat ~/.swiftlint-test.yml
custom_rules:
  545:
    regex: "(?:\S([ \t]|)+\{)"
% echo "if 545 {}"|swiftlint lint --config ~/.swiftlint-test.yml --use-stdin
Loading configuration from '/Users/norio/.swiftlint-test.yml'
<nopath>:1:6: warning: 545 Violation: Regex matched. (545)
Done linting! Found 1 violation, 0 serious in 1 file.
```

**Comment by jpsim** [comment]
Hopefully @norio-nomura answered your question!
